pontoon/api/schema.py

```diff
@@ -100,29 +100,40 @@ class Meta:
         )
 
     localizations = graphene.List(
-        ProjectLocale, include_disabled=graphene.Boolean(False),
+        ProjectLocale,
+        include_disabled=graphene.Boolean(False),
+        include_system=graphene.Boolean(False),
     )
 
-    def resolve_localizations(obj, _info, include_disabled):
+    def resolve_localizations(obj, _info, include_disabled, include_system):
         qs = obj.project_locale
 
-        if include_disabled:
+        if include_disabled and include_system:
             return qs.all()
+        elif include_disabled and not include_system:
+            return qs.filter(project__system_project=False)
+        elif not include_disabled and include_system:
+            return qs.filter(project__disabled=False)
 
-        return qs.filter(project__disabled=False)
+        return qs.filter(project__disabled=False, project__system_project=False)
```

Comment:
I think you can simplify this entire block to:
```python
return qs.filter(
    project__disabled=include_disabled,
    project__system_project=include_system,
)
```

Reply:
This is not giving correct result. Reason is QuerySet filter does AND on the condition. As per my understanding we need below result, correct?

// disabled = true and system = true - should return all the project
// disabled = true and system = false - should return all the project except system
// disabled = false and system = true - should return all the project except disabled
// disabled = false and system = false - should return all the project except disabled and system

I am not able to write complex query which will give this result with .filter/.exlude. Thus its better to keep this block.

Reply:
That's a great point!

I'm afraid the current solution doesn't scale very well. Imagine we'll add one or more conditions we'd like to exclude by default.

How about something like this?

```python
qs = obj.project_locale

records = qs.filter(disabled=False, system_project=False)

if include_disabled:
    records = records | qs.filter(project__disabled=True)
if include_system:
    records = records | qs.filter(project__system_project=True)

return records.distinct()
```